Doc/library/enum.rst

```diff
@@ -159,6 +163,22 @@ Data Types
 ----------
 
 
+.. class:: EnumDict
+
+   *EnumDict* is a subclass of :class:`dict` that keeps track of the order of enum members and prevents reusing member names. Use *EnumDict* when member names must be unique and their order needs to be preserved.
+
```

Comment:
Perhaps you can provide an example in a code block of when using `EnumDict` is different from a plain `dict`?

```suggestion
   *EnumDict* is a subclass of :class:`dict` that keeps track of
   the order of enum members and prevents reusing member names.
   Use *EnumDict* when member names must be unique
   and their order needs to be preserved.

   .. versionadded:: 3.14

```